frontend/src/metabase/admin/settings/components/SettingsSetting.jsx

```diff
@@ -9,7 +9,7 @@ import SettingNumber from "./widgets/SettingNumber";
 import SettingPassword from "./widgets/SettingPassword";
 import SettingRadio from "./widgets/SettingRadio";
 import SettingToggle from "./widgets/SettingToggle";
-import SettingSelect from "./widgets/SettingSelect";
+import SettingSelect from "./widgets/deprecated/SettingSelect";
```

Comment:
I see we don't name the deprecated component with any prefix, so I opt for a dedicated directory instead, so I don't have to think about making weird component names